### PR TITLE
Fixed iframe panel unnecessary scrollbar (on 100vh -64px)

### PR DESCRIPTION
**Pull request by sdotter**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @sdotter, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;frontend#5326).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by bramkragten** [comment]
Can you maybe explain what the problem was exactly and why this solves it?

**Comment by sdotter** [comment]
> Can you maybe explain what the problem was exactly and why this solves it?

Hi Bram! 
Yeah sure... this is my first attempt contribute to home assistant dev.

The issue here was that in an iframe panel... there is an unnecessary scrollbar.

See attached screenshots:

problem:
problem

fix:
fix

File: src/panels/iframe/ha-panel-iframe.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ class HaPanelIframe extends PolymerElement {
         iframe {
           border: 0;
           width: 100%;
+          margin-top: 0px;
```
Comment (bramkragten) [inline src/panels/iframe/ha-panel-iframe.js]: This is not needed

**Comment by bramkragten** [comment]
Thanks for the explanation, still not sure why it shows the scrollbars when the `position` isn't `absolute`

File: src/panels/iframe/ha-panel-iframe.js (hunk 1 above)
Comment (sdotter) [reply src/panels/iframe/ha-panel-iframe.js]: yep you're right... fixed it

**Review by bramkragten — APPROVED** [review]

**Pull request merged**